batcher/aligned-sdk/src/sdk.rs

```diff
@@ -153,10 +146,30 @@ pub async fn submit_multiple(
     .await
 }
 
+pub fn get_payment_service_address(network: Network) -> ethers::types::H160 {
+    match network {
+        Network::Devnet => H160::from_str("0x7969c5eD335650692Bc04293B07F5BF2e7A673C0").unwrap(),
+        Network::Holesky => H160::from_str("0x815aeCA64a974297942D2Bbf034ABEe22a38A003").unwrap(),
+        Network::HoleskyStage => {
+            H160::from_str("0x7577Ec4ccC1E6C529162ec8019A49C13F6DAd98b").unwrap()
+        }
+    }
+}
+
+pub fn get_aligned_service_manager_address(network: Network) -> ethers::types::H160 {
+    match network {
+        Network::Devnet => H160::from_str("0x1613beB3B2C4f22Ee086B2b38C1476A3cE7f78E8").unwrap(),
+        Network::Holesky => H160::from_str("0x58F280BeBE9B34c9939C3C39e0890C81f163B623").unwrap(),
+        Network::HoleskyStage => {
+            H160::from_str("0x9C5231FC88059C086Ea95712d105A2026048c39B").unwrap()
+        }
+    }
+}
```

Comment:
These addresses should probably not be hardcoded into the program.

Reply:
They are coded into the SDK, not the actual Batcher program.

This allows to ask the SDK directly for Aligned contract's addresses, so program implementations just need to pass in which `Network` they want to communicate.

The improvement is that before, these value were all around the place, expecting them in a parameter but then rewriting them according to other parameter values. (`payment_service_addr` and `chain`). Unifying this parameter (now called `Network`) avoids the possibility of sending uncompatible `payment_service_addr` and `chain` on the same request.

If the addresses where removed from the SDK, users would have to pass all relevant addresses as parameters, which would re-enable the possibility of sending uncompatible parameters on a request.

If necesarry, this could be done, but we should be extra careful with the error codes since uncompatible addresses might fail in unexpected ways, and error the error codes might not be obviously correlated to uncompatible addresses.

Reply:
I meant more that maybe they should be part of some config file. But I don't think that's really important at this time, so feel free to ignore the comment.